config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
```

Comment:
- [ ] [Optional] Kindly add a root route, You can make it like that:
```ruby
root "users#index"
```

Reply:
Ok, I had added it but in the end I hesitated to leave it but thanks for your comment. I'm going to add it.